_docs_integrate/data-model-overview.md

```diff
@@ -51,17 +51,29 @@ Tokens can be used to save arbitrary structured data on the Backbone, which is e
+| createdByDevice    | `string`                                                                                             | {% include descr_createdByDevice class="Token" %}                                                                                                                                             |                                               |
+| content            | `unknown`                                                                                            | The content of the Token. You can add whatever you want here.                                                                                                                                 | will be encrypted before sent to the Backbone |
+| createdAt          | `string`                                                                                             | {% include descr_createdAt class="Token" %}                                                                                                                                                   |                                               |
+| expiresAt          | `string`                                                                                             | {% include descr_expiresAt class="Token" %}                                                                                                                                                   |                                               |
+| forIdentity        | `string` \| `undefined`                                                                              | Can be set to an enmeshed address. If set, then only the [Identity]({% link _docs_integrate/data-model-overview.md %}#identity) with that `address` can retrieve the Token from the Backbone. |
+| passwordProtection | [`object for password protection of Token`](#object-for-password-protection-of-token) \| `undefined` | Information about whether the Token is protected by a password or not.                                                                                                                        |                                               |
+| truncatedReference | `string`                                                                                             | {% include descr_truncatedReference class="Token" %}                                                                                                                                          | saved only locally                            |
+| isEphemeral        | `boolean`                                                                                            | If set to `true` the Token will not be cached in the database and only displayed once. You will not be able to fetch this Token unless you remember its truncatedReference.                   |
+
+### Object for Password Protection of Token
```

Comment:
First of all, I understand your comment and that's why I initially considered it. I then decided that I don't want to change the title to `PasswordProtection` as long as the `class PasswordProtection` exists in the Transport library, because it is not exactly correct. Now renaming `class PasswordProtection` in the Transport library and typing `passwordProtection` in the runtime with PasswordProtection, just to embellish the title here, is beyond the scope of the PR in my opinion as I want to document the current code of the Runtime as correctly as possible within this PR.

Perhaps we could manage to turn the two descriptions into one description within this PR? However, I'm not sure how to do this if we don't have a defined object in the Runtime that could be meaningfully categorized in the Data Model Overview. So I would probably be in favor of leaving it as it is for now and then creating a ticket for refactoring in the Runtime and then also in the documentation.

Reply:
Resolved as we now have a PasswordProtectionDTO. (https://github.com/nmshd/runtime/pull/375)